view.html.erb

```diff
@@ -10,6 +10,7 @@
 
 <script type="text/javascript">
 (function () {
+  nonce="<%= content_security_policy_nonce %>"
```

Comment:
> In any case, I'd say get rid of it.
The `nonce` itself? The login won't work without it. I can move it to the `<script>` tag though.

Reply:
Why are we adding inline js just to have to move it out later?  Shouldn't we begin by not using inline js?

Reply:
nevermind. I see the injected ruby.

Reply:
> The nonce itself? The login won't work without it. I can move it to the <script> tag though.

Are you sure? I just removed it and it works for me. 

> Why are we adding inline js just to have to move it out later? Shouldn't we begin by not using inline js?

I don't follow you.

Reply:
I'm nor sure what either of you are referring to. If you attempt to login without that you will get a blank page since the CSP will complain about the `nonce`. Adding it got things working. The `nonce` is set server-side and sent back and it worked, but I'm open to doing whatever is decided.

Reply:
nevermind. I see the injected ruby.

Reply:
> Why are we adding inline js just to have to move it out later? Shouldn't we begin by not using inline js?

>> I don't follow you.

Oh I see you're meaning. Yea, we're removing inline js in OOD proper - but this is an app, which is a harder problem to solve. We need this javascript for codeserver to work so the alternative would be to build it into OOD and somehow have the OOD recognize that this app specifically needs it.

It's that we can't remove it from the app until OOD can support it directly and we're not even close to that yet.

Reply:
It worked? So strange, that did not work the last week without it. Did something change in the development environment?

Reply:
> It worked? So strange, that did not work the last week without it. Did something change in the development environment?

😆 probably coincidence. nonce's don't get sent anywhere - they're essentially a signature. They tell the browser you can safely execute any javascript with this signature - and no other. In any case, they aren't sent back to the server, so they're not part of a login/handshake type situation.

Reply:
I was referring to the CSP not letting the JS load because it did not have the right `nonce` and that it was generated on the server and returned, not the client then used. 

Anyyyyway, yeah this now seems to work without it so it's all mute. I'll remove it and continue to eye this app suspiciously.